cs/forms.texy

```diff
@@ -335,9 +349,8 @@ $form->addButton('raise', 'Zvýšit plat')
 \--
 
 
-
-addImage($name, $alt = NULL) .{toc: addImage()}
------------------------------------------------
+addImageButton($name, $alt = NULL) .{toc: addImage()}
+-----------------------------------------------------
```

Comment:
ImageButton is name of class, but method is simply addImage. (And yes, it is bad name.)